frame/readme.md

```diff
@@ -0,0 +1,17 @@
+# Ubuntu Frame Integration Tests
+
+## To run
+`python3 -m unittest`
+
+To run inside of a virtual X11 server:
+`xvfb-run python3 -m unittest`
```

Comment:
This doesn't work any more, but I'm not sure if we want it to :)

Reply:
Oops, it's accidentally broken when you don't specify `MIR_TEST_SERVER`